test/e2e/handler/conditions.go

```diff
@@ -171,3 +158,22 @@ func waitForDegradedPolicy(policy string) {
 func waitForPolicy(policy string, matcher GomegaMatcher) {
 	policyConditionsStatusForPolicyEventually(policy).Should(matcher, "should reach expected status at NNCP '%s', \n current enactments statuses:\n%s", policy, enactmentsStatusToYaml())
 }
+
+func cleanConditionMessageAndTimestamp(receivedConditions shared.ConditionList) shared.ConditionList {
```

Comment:
This function is quite general, is there a reason to name the parameter `receivedConditions` instead of just `conditions`?

Also it may be good to reflect that the parameter is a list in the function name - `cleanConditionsMessageAndTimestamp`, because the next function does that - `matchConditionsFrom`.
But these are a cosmetic things

Reply:
done